drivers/serial/Kconfig.ns16550

```diff
@@ -82,6 +82,13 @@ config UART_NS16550_PARENT_INIT_LEVEL
 	  only post kernel and hence such platforms the UART driver instance init
 	  should be invoked only post kernel in case parent node is PCI.
 
+config UART_NS16550_TI_K3
+	bool "Add support for NS16550 variant specific to TI K3 SoCs"
+	help
+	  Enabling this configuration allows the users to use the UART port in
+	  Texas Instruments K3 SoCs by enabling a vendor specific extended register
```

Comment:
Can we have @slpp95prashanth take a look at this and validate on BBAI-64 as well?